ee/server/service/scripts.go

```diff
@@ -15,7 +15,11 @@ import (
 )
 
 func (svc *Service) RunHostScript(ctx context.Context, request *fleet.HostScriptRequestPayload, waitForResult time.Duration) (*fleet.HostScriptResult, error) {
-	const maxPendingScriptAge = 5 * time.Minute // any script older than this is not considered pending anymore on that host
+	// any script older than this is not considered pending anymore on that host
+	// orbit stops the script after 5 minutes, so we add an additional minute to
+	// have wiggle room since the notification system might take a while to
+	// process the script.
+	const maxPendingScriptAge = 6 * time.Minute
```

Comment:
this used to be `30` seconds, and the API timeout `1` minute. In https://github.com/fleetdm/fleet/pull/15779 we set both to `5` minutes, which breaks some of the flows as the API timeout needs to be longer than the host timeout.

I think `30` seconds extra might be enough but I added an extra minute. Very open to thoughts